pyracf/common/segment_trait_error.py

```diff
@@ -0,0 +1,18 @@
+"""Exception to use when the user passes bad segment-trait name(s) on an add/alter request."""
+
+
+class SegmentTraitError(Exception):
+    """
+    Raised when a user passes a bad segment-trait combination in the traits dictionary.
+    """
+
+    def __init__(self, bad_traits: list, profile_type: str) -> None:
+        self.message = "Unable to build Security Request.\n\n"
+        for trait in bad_traits:
+            self.message += (
+                f"'{trait}' is not a known segment-trait "
```

Comment:
"not a valid segment-trait"

Reply:
after discussion, we are keeping as not a known segment-trait since the user can update the list of "known" segments-traits